laravel/resources/views/pages/admin/user-list.blade.php

```diff
@@ -1,19 +1,33 @@
-<?php
-
-use App\Helpers;
```

Comment:
Removing this line is causing a fatal error.

```
[2019-07-17 00:31:53] laravel.ERROR: Class 'Helpers' not found (View: /home/voldb/laravel-voldb/laravel/resources/views/pages/admin/user-list.blade.php) {"userId":1,"email":"rachel@wetfish.net","exception":"[object] (ErrorException(code: 0): Class 'Helpers' not found (View: /home/voldb/laravel-voldb/laravel/resources/views/pages/admin/user-list.blade.php) at /home/voldb/laravel-voldb/laravel/storage/framework/views/32d208b491a03f35f775ccc8564690eb2a2dad6c.php:50, Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Class 'Helpers' not found at /home/voldb/laravel-voldb/laravel/storage/framework/views/32d208b491a03f35f775ccc8564690eb2a2dad6c.php:50)
```

Reply:
yeah I dunno what happened there tbh